crates/core_simd/src/vector.rs

```diff
@@ -100,14 +102,13 @@ use crate::simd::{
 // avoided, as it will likely become illegal on `#[repr(simd)]` structs in the future. It also
 // causes rustc to emit illegal LLVM IR in some cases.
 #[repr(simd, packed)]
+#[rustc_simd_monomorphize_lane_limit = "64"]
```

Comment:
The relevant test verifying failure on too-long values is here: https://github.com/rust-lang/rust/blob/a00a5159cfac23e9b1551566cf71ff31edc1f11b/tests/ui/simd/monomorphize-too-long.rs